### PR TITLE
Fix issue #3 and #4

### LINKED ISSUE #3: Ethernet instable, kind of deadlocking the controller; Nucleo F429ZI
I'm expiriencing some issues while using the F429ZI as a http-server. (quite simple application, where a few controller pins are set according to http requests from clients)

It generally does what it is meant to do, but at some point clients can not reach the server any more. If client is a web browser, opening a new tab solves this temporarely, but soon or later the controller gets kind of locked, meaning it is completly unreachable, and even code in main loop does not seem to get executed any more (tested by attaching switch on one pin, which turns on/off another GPIO).

There does not seem to be a specific number of requests or clients leading to failure, sometimes two or three requests from one single client already lead to a complete lock, sometimes it takes a few thousands from different clients, but at some point it always fails. 
One strange thing is, that requests from webbrowser seem to lead to failure way faster, than from other applications like Curl, especially when multiple clients are involved. Maybe some timing issue??

The behavior can be reproduced with the server-example from the library. Just starting the server and then have for example five or six Curl clients throwing requests at it every few seconds, makes it fail quite fast most of the time, especially when also sending requests from a webbrowser manually.
I used this simple bat script for testing:

```
for /L %%N IN (1, 1, 10000) DO (
echo Nummer %%N
t:\curl.exe 10.66.22.124
ping -n 2 127.0.0.1 > NUL 
)
```
I hope there is someone who can also reproduce this and is familiar with the ethernet implementation, because I absolutely have no clue how to track this down...

### PR DESCRIPTION
**Pull request by fprwi6labs**
This PR should fix #3 
- Ethernet scheduler called by timer every 1ms
- function close client reworked

To test this PR please add `#define DEFAULT_ETHERNET_TIMER TIM6` in lwipopts.h

The issue #4 is also fixed:
- init sequence is done only once even if `begin()` is called several time.

### CODE REVIEW

File: src/utility/stm32_eth.c [hunk 1]
```diff
@@ -132,6 +136,34 @@ static void Netif_Config(void)
...
+/**
+* @brief  Enable the timer used to call ethernet scheduler function at regular
+*         interval.
+* @param  None
+* @retval None
+*/
+static void TIM_scheduler_Config(void)
+{
+  /* Set TIMx instance. */
+  TimHandle.timer = DEFAULT_ETHERNET_TIMER;
```
[the remark this answers is not in the document]
Comment (fprwi6labs) [reply src/utility/stm32_eth.c]: I defined it in lwipopt.h. I think it is better to keep all Ethernet options in this file.

**Comment by fprwi6labs** [comment]
If this PR is validated, the register callback could be removed indeed.

**Comment by fpistm** [comment]
please @fprwi6labs could you remove the register cb.

**Comment by fprwi6labs** [comment]
Please merge also:
- in core [PR#136](https://github.com/stm32duino/Arduino_Core_STM32/pull/136)
and
- LwIP [PR#2](https://github.com/stm32duino/LwIP/pull/2)

**Comment by fprwi6labs** [comment]
Issue #4 fixed. Initialization sequence must be done only once.

**Pull request merged**